package lpfx.desafio.exceptions;

public class RecursoNaoEncontrado extends RuntimeException {

}
